projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -248,6 +248,11 @@ spec:
             - name: DATAJOBS_READ_ONLY_ROOT_FILESYSTEM
               value: "{{ .Values.dataJob.readOnlyRootFilesystem }}"
             {{- end }}
+            {{- if .Values.deploymentBuilderImage.password }}
```

Comment:
The if condition here and in the secret_pull_builder_img.yaml are different (username vs password). 

That seems like the possibility of a unexpected behaviour e.g somewhere where password is empty .

Reply:
Resolved

Reply:
If I set password but do not set user. then the secret won't be created but this will be created and the deployment would fail ? a